packages/apps/tools/src/constants/kadena.ts

```diff
@@ -1,41 +1,46 @@
 import { env } from '@/utils/env';
+import type { ChainwebNetworkId } from '@kadena/chainweb-node-client';
+
+export type NetworkNames = Exclude<ChainwebNetworkId, 'development'>;
+export type NetworkName = NetworkNames | string;
 
-export type DefinedNetwork = 'mainnet01' | 'testnet04';
-export type Network = DefinedNetwork | string;
 export type DevOption = 'BASIC' | 'BACKEND' | 'DAPP';
 
-type KadenaConstants = {
-  [K in DefinedNetwork]: {
-    label: string;
-    API: string;
-    apiHost: (params: { networkId: string; chainId: string }) => string;
-    estatsHost: () => string;
-  };
-} & {
+interface KadenaConstants {
   GAS_LIMIT: number;
   GAS_PRICE: number;
   API_TTL: number;
   DEFAULT_SENDER: string;
-};
+}
+
+interface NetworkType {
+  label: string;
+  API: string;
+  apiHost: (params: { networkId: string; chainId: string }) => string;
+  estatsHost: () => string;
+}
 
 export const kadenaConstants: KadenaConstants = {
+  GAS_LIMIT: Number(env('GAS_LIMIT', 850)),
+  GAS_PRICE: Number(env('GAS_PRICE', 0.00000001)),
+  API_TTL: Number(env('KADENA_API_TTIL', 600000)),
+  DEFAULT_SENDER: env('DEFAULT_SENDER', 'not-real'),
+};
+
+export const kadenaDefaultNetworks: Record<NetworkNames, NetworkType> = {
```

Comment:
This would then become `ChainwebNetworkId`
```suggestion
export const kadenaDefaultNetworks: Record<ChainwebNetworkId, NetworkType> = {
```